projects/avataaars-angular/src/public-api.ts

```diff
@@ -2,6 +2,13 @@
  * Public API Surface of avataaars-angular
  */
 
+
+
 export * from './lib/avataaars-angular.service';
-export * from './lib/avataaars-angular.component';
+export * from './lib/components/avatar/avatar.component';
 export * from './lib/avataaars-angular.module';
+
+
+// enums
```

Comment:
lets get rid of this commented line and reduce some white space lines here

Reply:
Done